synapse/lib/ast.py

```diff
@@ -2866,6 +2866,9 @@ class FuncCall(Value):
     async def compute(self, runt, path):
 
         func = await self.kids[0].compute(runt, path)
+        if not callable(func):
+            raise s_exc.StormRuntimeError(mesg=f"'{type(func).__name__}' object is not callable.")
```

Comment:
Would we want to try to map these to storm types rather than allowing python types to slip through?

Reply:
updated to use `totype`